### ISSUE TITLE
Failed to apply default options, could not create directory

### ISSUE THREAD

**Issue by sargunv** [shedaniel/your-options-shall-be-respected#1]
```
[03:16:43] [main/ERROR]: Failed to apply default options.
java.lang.IllegalStateException: Could not create directory: C:\Program Files\MultiMC\instances\1.15.2 fabric personal pack\.minecraft\config\C:\Program Files\MultiMC\instances\1.15.2 fabric personal pack\.minecraft\config\yosbr\config\i-need-keybinds
```

full log is here: https://paste.ee/p/aFuop

looks like it's appending an absolute path to the base instead of a relative path

**Comment by shedaniel** [comment]
Looks like it, I will fix this.

**Comment by shedaniel** [comment]
Fixed in v0.1.1
